implementations/rust/ockam/ockam_command/src/authenticated.rs

```diff
@@ -95,45 +84,49 @@ async fn make_background_node_client(
     Ok(BackgroundNode::create(ctx, &opts.state, &node_name).await?)
 }
 
-fn print_entries(entries: &[(Identifier, AttributesEntry)]) {
-    let template = TextTemplate::from(LIST_VIEW);
-    let model: Vec<_> = entries
-        .iter()
-        .map(|(identifier, entry)| {
-            let attrs: HashMap<String, String> = entry
-                .attrs()
-                .iter()
-                .map(|(k, v)| {
-                    (
-                        String::from_utf8(k.clone()).unwrap(),
-                        String::from_utf8(v.clone()).unwrap(),
-                    )
-                })
-                .collect();
-            (
-                String::from(identifier),
-                serde_json::to_string(&attrs).unwrap(),
-                format!("{:?}", entry.added()),
-                entry
-                    .expires()
-                    .map_or("-".to_string(), |t| format!("{:?}", t)),
-                entry.attested_by().map_or("-".to_string(), String::from),
-            )
-        })
-        .collect();
-    let mut expander = template.expander();
+struct IdentityEntry<'a> {
+    identifier: &'a Identifier,
+    entry: &'a AttributesEntry,
+}
 
-    model.iter().for_each(
-        |(identifier, attributes, created_at, expires_at, attested_by)| {
-            expander
-                .sub("identity")
-                .set("identifier", identifier)
-                .set("attributes", attributes)
-                .set("created_at", created_at)
-                .set("expires_at", expires_at)
-                .set("attested_by", attested_by);
-        },
-    );
-    let skin = MadSkin::default();
-    skin.print_expander(expander);
+impl<'a> Display for IdentityEntry<'a> {
```

Comment:
We want to implement the `Output` trait here, so you can print it like in the example from the issue description: https://github.com/build-trust/ockam/blob/da8e3a5e109ed2ed265cdbac05a9f9e890e21451/implementations/rust/ockam/ockam_command/src/lease/list.rs#L57